src/commons/utils/corio_utils.py

```diff
@@ -269,6 +279,52 @@ def setup_environment():
     LOGGER.info("environment setup completed.")
 
 
+def activate_degraded_mode(options: dict) -> None:
+    """Create bucket and degrade node as needed .
+
+    :param options : dictionary to fetch command line params
+    :return : None
+    """
+    bucket = f'object-op-{time.perf_counter_ns()}'.lower()
+    response = S3Bucket(access_key=options.access_key,
+                        secret_key=options.secret_key,
+                        endpoint_url=options.endpoint_url,
+                        use_ssl=options.use_ssl).create_bucket(bucket)
+    if response:
+        os.environ["d_bucket"] = bucket
+
+    if options.degrade_pod:
+        __degrade_nodes(options)
+
+
+def __degrade_nodes(options: dict) -> None:
```

Comment:
This methods should be part of separate lib as corio_utils is common modules having common methods from tool.

Reply:
Refactored accordingly.